test/WebJobs.Extensions.RabbitMQ.Samples/RabbitMQTriggerSamples.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Json;
+using System.Text;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.RabbitMQ;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Extensions.Logging;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+
+namespace WebJobs.Extensions.RabbitMQ.Samples
+{
+    public static class RabbitMQTriggerSamples
+    {
+        public static void RabbitMQTrigger_String(
+            [RabbitMQTrigger("localhost", "queue")] string message,
+            string consumerTag,
+            ILogger logger
+        )
+        {
+            logger.LogInformation($"RabbitMQ queue trigger function processed message consumer tag: {consumerTag}");
+            logger.LogInformation($"RabbitMQ queue trigger function processed message: {message}");
+        }
+
+        public static void RabbitMQTrigger_BasicDeliverEventArgs(

```

Comment:
Add another sample that uses RabbitMQTrigger and RabbitMQ output binding- listening on one queue and writing to a different queue